src/scss/base/_seo.scss

```diff
@@ -1,15 +1,10 @@
 [data-seo-container] {
     position: relative;
 
-    &:hover,
-    &:focus-within {
+    &:hover {
         cursor: pointer;
     }
```

Comment:
@Cocola inutile selon moi de préciser le cursor: pointer; au hover, on peut le mettre directement sur [data-seo-container] puisque, of course, le champ de curseur se verra au hover.